internal/pkg/workspace/workspace.go

```diff
@@ -60,13 +61,13 @@ const (
 // Summary is a description of what's associated with this workspace.
 type Summary struct {
 	Application string `yaml:"application"` // Name of the application.
-	Path        string `yaml:"-"`           // absolute path to the summary file.
+	Path        string `yaml:"-"`           // Absolute path to the summary file.
 }
 
 // Workspace typically represents a Git repository where the user has its infrastructure-as-code files as well as source files.
 type Workspace struct {
 	workingDirAbs string
-	copilotDirAbs string
+	CopilotDirAbs string
```

Comment:
for unit testing in the selector package :(

Reply:
ic!  I think that indicates that we need to `mock` the `ProjectRoot` call in `local_selector_test.go`, instead of using a real  `workspace.Workspace`. I am not sure if 👇🏼 is doable, so I'm fine if we prefer keeping it exported as for now (but we probably need to refactor it later so that we don't need to export a var for another package's testing).
```go
type workspaceRootReader interface {
  ProjectRoot() string
}
type LocalFilesSeletor struct {
  ws workspaceRootReader
}

func TestLocalFileSelector_listDirsAndFiles(t *testing.T) {
	testCases := map[string]struct {
		mockFileSystem func(mockFS afero.Fs)
                 mockWS func(m *mocks.workspaceRootReader)
		wantedErr      error
		dirsAndFiles   []string
	}{
		"drill down two (and only two) levels": {
                         mockWS: func(m *mocks.workspaceRootReader) {
                               m.Expect().ProjectRoot().Return("mockRott")
                         }
                 }
         }
```